frontend/src/visualization/shared-ops/entityStyle.ts

```diff
@@ -0,0 +1,34 @@
+import { EdgeStyle, NodeStyle } from '../../stores/colors';
+
+type NodeStyleProperties = {
+  color: { border: string; background: string };
+  borderWidth: number;
+  shapeProperties: { borderDashes: boolean | number[] };
+};
+
+type EdgeStyleProperties = {
+  color: string;
+  dashes: boolean | number[];
+  width: number;
+};
```

Comment:
This type already exists.

```suggestion
import {
  EdgeOptions,
  NodeOptions,
} from 'vis-network/declarations/network/Network';
```